kratos/linear_solvers/linear_solver.h

```diff
@@ -158,6 +158,7 @@ class LinearSolver
     */
     virtual void PerformSolutionStep(SparseMatrixType& rA, VectorType& rX, VectorType& rB)
     {
+        Solve(rA, rX, rB); //call standard solve function as fallback
```

Comment:
@KratosMultiphysics/technical-committee  
I added this line in case a linear solver that does not implement those three functions:

1. ```InitializeSolutionStep```
2. ```PerformSolutionStep```
3. ```FinalizeSolutionStep```

can still be used everywhere